packages/dx-react-grid-demos/src/demo-sources/grid-virtual-scrolling/scroll-to-row.jsxt

```diff
@@ -34,6 +34,14 @@ export default () => {
     { columnName: 'id', width: 75 },
   ]);
   const vtRef = useRef();
+  const [defaultSorting] = useState([
+    { columnName: 'name', direction: 'asc' },
+  ]);
```

Comment:
I think we should define `defaultXXX` properties as a constant instead of `react state` because these properties are not reactive and these changes will not rerender an element.

Reply:
The same for `sortingStateColumnExtensions`?

Reply:
If it is a `reactive` property